Landing-page.css

```diff
@@ -0,0 +1,842 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;600;800&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #fff;
+    font-family: Nunito;
+    overflow-x: none;
+}
+
+.container {
+    width: 90vw;
+    margin: auto;
+
+}
+
+ul {
+    list-style-type: none;
+}
+
+/**
+** HEADER
+**/
+
+header {
+    display: flex;
+    padding-top: 2.6rem; 
+}
+
+header .header-lt {
+    
+}
+
+header .header-lt h2 {
+    color: #c4c4c4;
+}
+
+header .header-rt {
+    /*
+    display: flex;
+    justify-content: space-between;
+    width: 200px;
+    */
+    display: flex;
+    justify-content: space-around;
+    margin-left: auto;
+}
+
+.search-box {
+    display: flex;
+    align-items: center;
+    border-radius: 2rem;
+}
+
+.search-box svg {
+    height: 2.3rem;
+    width: 2.3rem;
+    fill: #1F59BB;
+}
+
+
+.search-box input {
+    border: none;
+    background: none;
+    width: 0;
+    outline: none;
+}
+
+.search-box a {
+    padding: 0.5rem;
+    background: rgb(93, 121, 186);
+    border-radius: 10rem;
+}
+
+.search-box:hover {
+    padding-left: 2rem;
+    padding-right: 1rem; 
+    background:  rgba(53, 151, 245, 0.1);
+    border: .05rem solid #fff;
+    fill: #1E1666;
+}
+
+.search-box:hover input {
+    width: 30vw;
+}
+
+.search-box:hover a {
+    color: #fff;
+}
+
+.search-box:hover svg {
+    fill: #1F59BB;
+}
+
+.search-box,
+.search-box a,
+.search-box input {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected selector ".search-box input" to come before selector ".search-box:hover input" (no-descending-specificity)](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5575014)